tmux/task_monitor_debug.yml

```diff
@@ -71,6 +71,7 @@ windows:
       -r __ns:=${ROS_NAMESPACE}
       -p det_topic:=ActivityDetections
       -p task_update_topic:=TaskUpdates
+      -p task_steps:=${CONFIG_DIR}/default_task_steps.json
```

Comment:
This config should more specifically reference that it is the MIT-LL eval 1 coffee recipe steps instead of "default".